viam_cartographer.go

```diff
@@ -34,6 +35,10 @@ var (
 	ErrClosed = errors.Errorf("resource (%s) is closed", Model.String())
 	// ErrUseCloudSlamEnabled denotes that the slam service method was called while use_cloud_slam was set to true.
 	ErrUseCloudSlamEnabled = errors.Errorf("resource (%s) unavailable, configured with use_cloud_slam set to true", Model.String())
+	// ErrNoPostprocessingToUndo denotes that the points have not been properly formatted.
+	ErrNoPostprocessingToUndo = errors.New("there are no postprocessing tasks to undo")
+	// ErrBadPostprocessingPointsFormat denotest that the postprocesing points have not been correctly provided.
+	ErrBadPostprocessingPointsFormat = errors.New("invalid postprocessing points format")
```

Comment:
same with these and the constants in 53-57, if they can be unexported, that's best.

Reply:
i can't use them in the viam cartographer tests if they are not exported

Reply:
ah, since it is a `_test` package. Boooo